cli/azd/pkg/environment/environment.go

```diff
@@ -144,13 +139,38 @@ func (e *Environment) Getenv(key string) string {
 	return os.Getenv(key)
 }
 
+// Reloads environment variables from .env file
+func (e *Environment) Reload() error {
+	envPath := filepath.Join(e.Root, azdcontext.DotEnvFileName)
+	if envMap, err := godotenv.Read(envPath); errors.Is(err, os.ErrNotExist) {
+		e.Values = make(map[string]string)
+	} else if err != nil {
+		return fmt.Errorf("loading .env: %w", err)
+	} else {
+		e.Values = envMap
+	}
```

Comment:
```suggestion
	if e.Values, err := godotenv.Read(envPath); errors.Is(err, os.ErrNotExist) {
		e.Values = make(map[string]string)
	} else if err != nil {
		return fmt.Errorf("loading .env: %w", err)
	}
```

Reply:
Coped from original load version above.  Going to keep this for now since it's working.